test/common/test_centroid.cpp

```diff
@@ -61,26 +61,30 @@ TEST (PCL, compute3DCentroidFloat)
   std::vector<int> indices;
   PointXYZ point;
   PointCloud<PointXYZ> cloud;
-  Eigen::Vector4f centroid;
+  Eigen::Vector4f centroid = Eigen::Vector4f::Ones ();
```

Comment:
It'd be better to do one of the following:
* have another variable say, `old_centroid` with which comparison is performed
* a note stating that `Eigen::Vector4f::Ones ()` isn't special
* comment in other places (like L69, 179) saying centroid should be unchanged

Reply:
fixed

Reply:
If Eigen::Vector4f::Ones () isn't special, why not just leave uninitialized?